src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,33 @@
+import s from './HomePage.module.scss';
+import WorldMap from '../../../public/img/World Map.svg';
+import Button from '../Button/Button';
+
+export const HomePage = () => {
+  return (
+    <div className={s.container}>
+      <div className={s.titleContainer}>
+        <h5 className={s.title}>
+          Empower with <br />
+          <h5 className={s.subtitle}>AQE Educate.</h5>
+        </h5>
+
+        <p className={s.paragraph}>
+          Your trusted partner in professional development <br /> for educators.
+        </p>
+      </div>
+      <img className={s.img} src={WorldMap} alt="worldMap" />
```

Comment:
image is misaligned 
<img width="722" alt="Zrzut ekranu 2024-07-25 o 15 33 16" src="https://github.com/user-attachments/assets/dbaf72c8-f85a-4087-ad5d-fc9de9d0d4ad">
<img width="667" alt="Zrzut ekranu 2024-07-25 o 15 33 24" src="https://github.com/user-attachments/assets/60818580-e74f-4298-91a6-2c611d02ba35">